dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,331 @@
+package dbfit.environment;
+
+import dbfit.annotations.DatabaseEnvironment;
+import dbfit.api.AbstractDbEnvironment;
+import dbfit.fixture.StatementExecution;
+import dbfit.util.DbParameterAccessor;
+import dbfit.util.DdlStatementExecution;
+import dbfit.util.Direction;
+import dbfit.util.NameNormaliser;
+import dbfit.util.Options;
+import dbfit.util.TypeNormaliserFactory;
+import fit.TypeAdapter;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+
+@DatabaseEnvironment(name="Informix", driver="com.informix.jdbc.IfxDriver")
+public class InformixEnvironment extends AbstractDbEnvironment {
+
+    @Override
+    public void afterConnectionEstablished() throws SQLException {
+        if (currentConnection.getMetaData().supportsTransactions()) {
```

Comment:
Do you mean *how will the driver respond to that*?

The `CommitAndRollback` test gets run in the `ANSI` mode test suite for Informix and passes.